app/src/main/java/com/concordium/wallet/data/walletconnect/Schema.kt

```diff
@@ -0,0 +1,6 @@
+package com.concordium.wallet.data.walletconnect
+
+data class Schema (
+    val type: String?,
+    val value: String?,
```

Comment:
Shouldn't these be non-nullable?

Reply:
GSON is a java serialisation/deserialisation library that is used in this project, but there is an issue in that it doesn't recognise Kotlin's non-nullable types. So if I state them as not nullable in Kotlin, deserialising them with GSON can still produce null and issue later on.